runners/remote-worker/src/lib/credhelper.ts

```
@@ -84,17 +84,10 @@ export function credHelperScript(params: CredHelperParams): string {
-#   (a) publisher cc — when PUBLISHER_CLIENT_ID/SECRET/TOKEN_URL are set,
-#       mint a Thunder access token via client_credentials.
-#   (b) legacy TaskJWT — read the per-task bearer from \$AEP_BEARER_FILE.
-#
-# \$AEP_BEARER_FILE is supplied by whoever invokes git, which is what lets the
-# same bytes serve both phases of a run: the provisioning clone points it at a
-# staged bearer (the workspace does not exist yet), and the agent's environment
-# points it at .aep/bearer. The script is never rewritten between the two.
+# Authenticates to the platform with the org publisher client_credentials token
+# (POST /internal/v1/executions/{executionId}/credentials/refresh). When
+# PUBLISHER_CLIENT_ID/SECRET/TOKEN_URL are set, mint a Thunder access token.
+# If that mint fails, read the CC snapshot oneshot wrote to \$AEP_BEARER_FILE.
```

ESLint

[error] Unnecessary escape character: \$.

(no-useless-escape)